### PR TITLE
Dropdown visibility

### LINKED ISSUE #154: MoreOptionsDropdown does not appear if the user is not login.
### Describe the bug

`MoreOptionsDropdown` does not appear if the user is not login, so they can't share posts without login.

### Steps to reproduce the behavior

Check the post list without login.

### Expected behavior

`MoreOptionsDropdown` appears with a share option even if user is logged out.

[screen-capture.webm](https://user-images.githubusercontent.com/22305845/202824237-b1c7ed18-68e0-4f2f-932c-027dbbf715b6.webm)

### PR DESCRIPTION
**Pull request by csoylu**
fixes #154
Post item used to check if user was able to edit the post to show the dropdown. I removed it.
A check for userCanEdit already exists in dropdown, fixed some UI.

There's also a commit on prisma seed that can make multiple users.

### CODE REVIEW

File: apps/pano/src/features/post/MoreOptionsDropdown.tsx [hunk 1]
```diff
@@ -84,7 +80,9 @@ export const MoreOptionsDropdown: FC<Props> = ({ post }) => {
         </DropdownMenuTrigger>
         <DropdownMenuContent>
           {ownerItems}
-          <DropdownMenuSeparator />
+          {canUserEdit(user, post) && (
+            <DropdownMenuSeparator />
+          )}
```
Comment (usirin) [inline apps/pano/src/features/post/MoreOptionsDropdown.tsx:85]: Let's move this to `ownerItems`

**Review by usirin — CHANGES_REQUESTED** [review]

File: apps/pano/prisma/seed.ts [hunk 2]
```diff
@@ -1,36 +1,45 @@
 import { PrismaClient } from "@prisma/client";
 import bcrypt from "bcryptjs";
 
+const users = [
+  { username: "admin", email: "admin@kamp", password: "123123123" },
+  { username: "testuser", email: "user1@kamp", password: "123123123" }
```
Comment (usirin) [inline apps/pano/prisma/seed.ts:6]: use valid email addresses please

File: apps/pano/src/features/post/MoreOptionsDropdown.tsx (hunk 1 above)
Comment (csoylu) [reply apps/pano/src/features/post/MoreOptionsDropdown.tsx]: done

File: apps/pano/src/features/post/MoreOptionsDropdown.tsx [hunk 3]
```diff
@@ -32,6 +32,9 @@ const DotsButton = styled(IconButton, {
 
 const Item = styled(DropdownMenuItem, {
   minWidth: 0,
+  "&:hover": {
+    cursor: "pointer",
+  },
```
Comment (usirin) [inline apps/pano/src/features/post/MoreOptionsDropdown.tsx:37]: Let's revert this, menu interactions don't need to have `cursor: pointer`

Comment (csoylu) [reply apps/pano/src/features/post/MoreOptionsDropdown.tsx]: done

**Review by usirin — APPROVED** [review]
💯

**Pull request merged**